contrib/update/requirements.txt

```diff
@@ -0,0 +1,2 @@
+ansible==2.7.5
+git+https://github.com/srkiNZ84/dopy.git@master  # until dopy supports Python 3
```

Comment:
That's I'm saying, you can have Ansible destroying your droplet, the key doesn't have to be destroyed (if you use the same key every time). Then you can completely ditch this non-official dependency. Here you rely on `srkiNZ84` won't add any malicious code that will have access to your DO credentials.

Reply:
Python 3 issue is not related to the key issue. It's related to the Docker image used in the container I installed Ansible. The point here is Python 2 end of life (scheduled for this year) so I would like to push for Python 3. Either way, we do not have to depend on a wrapper as DigitalOcean API canbe used with HTTP requests.

Reply:
Fixed by f6cb340